MEDO v0.5.owl

```diff
@@ -804,6 +975,536 @@ LCME Glossary of Terms for LCME Accreditation Standards and Elements 2015-2016,
+
+Does not include: Clinical Documentation Review; Clinical Performance Rating/Checklist; Peer Assessment; Self-Assessment</obo:IAO_0000119>
+        <rdfs:label>formal summative teacher observation and feedback</rdfs:label>
+        <skos1:related>MedBiquitous - AM10: Narrative Assessment</skos1:related>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/medo.owl#informal_formative_oral_patient_presentation -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/medo.owl#informal_formative_oral_patient_presentation">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/MEDO_0000106"/>
+        <obo:IAO_0000115>A type of informal formative assessment where a student presents clinical case (patient) findings, history and physical, differential diagnosis, treatment plan, etc., to an instructor or small group for the purpose of demonstrating skills in clinical reasoning and problem-solving and discussing the experience, process, and performance.</obo:IAO_0000115>
+        <obo:IAO_0000119>MedBiquitous - AM11: The presentation of clinical case (patient) findings, history and physical, differential diagnosis, treatment plan, etc., by a learner to an instructor or small group, and subsequent discussion with the instructor and/or small group for the purposes of learner demonstrating skills in clinical reasoning, problem-solving, etc. (Wiener, 1974)</obo:IAO_0000119>
+        <rdfs:label>informal formative oral patient presentation</rdfs:label>
+        <skos1:related>MedBiquitous - AM11: Oral Patient Presentation</skos1:related>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/medo.owl#informal_formative_participation_assessment -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/medo.owl#informal_formative_participation_assessment">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/MEDO_0000106"/>
+        <obo:IAO_0000115>A type of informal formative assessment that evaluates the student&apos;s attendance, participation, engagement, and performance in an activity.</obo:IAO_0000115>
+        <obo:IAO_0000119>MedBiquitous - AM12: Sharing or taking part in an activity (ERIC, 1966b) 
+
+Synonymous with/Includes: Attendance</obo:IAO_0000119>
+        <rdfs:label>informal formative participation assessment</rdfs:label>
+        <skos1:related>MedBiquitous - AM12: Participation</skos1:related>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/medo.owl#informal_formative_peer_assessment -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/medo.owl#informal_formative_peer_assessment">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/MEDO_0000106"/>
+        <obo:IAO_0000115>A type of informal formative assessment where fellow learner(s) observes, assesses, and provides written or oral subjective feedback of a learner&apos;s work or performance.</obo:IAO_0000115>
```

Comment:
My revision "A type of informal formative assessment in which one or more fellow learners observe, assess, and provide written or oral subjective feedback of a learner's work or performance."